yt_dlp/extractor/youtube.py

```diff
@@ -2503,16 +2506,15 @@ def _extract_player_responses(self, clients, video_id, webpage, master_ytcfg, pl
             if pr:
                 yield pr
 
-            if self._is_agegated(pr):
-                if client.endswith('_agegate'):
-                    # _creator clients can bypass AGE_VERIFICATION_REQUIRED if logged in
-                    if not self._generate_sapisidhash_header():
-                        continue
-                    client = '%s_creator' % client.remove_end('_agegate')
-                else:
-                    client = f'{client}_agegate'
-                if client in INNERTUBE_CLIENTS and client not in original_clients:
-                    clients.append(client)
+            def append_client(client_name):
```

Comment:
That's how it works for me. Just a suggestion. Try creator client if age-gate bypass fails with `UNPLAYABLE` and cookies are given. (3.)

Non embeddable video without Cookies
```
[youtube] Cr381pDsSsA: Downloading webpage
[youtube] Cr381pDsSsA: Downloading android player API JSON
[youtube] Cr381pDsSsA: Downloading android agegate player API JSON
[youtube] Cr381pDsSsA: Downloading web agegate player API JSON
ERROR: Playback on other websites has been disabled by the video owner.
```

Non embeddable video + Cookies
```
[youtube] Cr381pDsSsA: Downloading webpage
[youtube] Cr381pDsSsA: Downloading android player API JSON
[youtube] Cr381pDsSsA: Downloading android agegate player API JSON
[youtube] Cr381pDsSsA: Downloading android creator player API JSON
[youtube] Cr381pDsSsA: Downloading web agegate player API JSON
[youtube] Cr381pDsSsA: Downloading web creator player API JSON
[info] Cr381pDsSsA: Downloading 1 format(s): 22
```

Embeddable video without Cookies:
```
[youtube] 7t0SqerlBA0: Downloading webpage
[youtube] 7t0SqerlBA0: Downloading android player API JSON
[youtube] 7t0SqerlBA0: Downloading android agegate player API JSON
[youtube] 7t0SqerlBA0: Downloading web agegate player API JSON
[info] 7t0SqerlBA0: Downloading 1 format(s): 22
```